index.js

```diff
@@ -56,14 +58,34 @@ function groups (groups, data, aggregations, thisArg, additionalArgs) {
   data = Array.isArray(data) ? data : data.features
   var args = [undefined, undefined, undefined].concat(additionalArgs)
 
+  var tree = rbush(data.length)
```

Comment:
> Higher value means faster insertion and slower search, and vice versa.

With 1000 features this actually makes it a little faster. My guess is that more features will make smaller branching factors better